File: RedSheet.DbEntities/Models/vNanoTheirBatna.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vNanoTheirBatnas", Schema = "dbo")]
	[Serializable]
	public partial class vNanoTheirBatna  
	{
 
	[DatabaseGenerated(DatabaseGeneratedOption.Identity)] 
	[Key]
	public int NanoTheirBatnaId { get; set; } 
 
	[Required] 
	[MaxLength(1000)]
	public string NanoTheirBatnaValue { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int ProjectModuleId { get; set; } 

	public vNanoTheirBatna()
	{
    }

    }
}

File: redsheet-portal/src/app/database-models/v-nano-their-batna.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vNanoTheirBatna {
    constructor(vNanoTheirBatna?: vNanoTheirBatna )  {
        let properties = [ "nanoTheirBatnaId", "nanoTheirBatnaValue", "projectModuleId",];
        for (let property of properties)
            if (vNanoTheirBatna && vNanoTheirBatna[property])
                this[property] = vNanoTheirBatna[property];
    }
 
	nanoTheirBatnaId : number =   0 ;
 
    @required()
    @maxLength(1000)
	nanoTheirBatnaValue : string =   undefined;
 
    @range(1,2147483647)
	projectModuleId : number =   undefined;


}
